### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ce8m4ayqjn3tc3y05ka9p6qmq0

[download apk](http://filestorage.bluewallet.io:1488/1592329266.apk)

File: README.md [hunk 1]
```diff
@@ -69,14 +69,14 @@ The above command will build the app and install it. Once you launch the app it
 
 ```
 cd ios
-pod install
+npx pod-install
```
Comment (Overtorment) [inline README.md:72]: our scripts dont have `pd-install`, but the do have `podinstall` (you would not need `cd` in that case)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8pv7ma9nwtdywcyunmj3bv3ryg

[download apk](http://filestorage.bluewallet.io:1488/1595426130.apk)

File: README.md (hunk 1 above)
Comment (marcosrdz) [reply README.md:71]: Fixed

**Pull request merged**